vllm/v1/engine/async_llm.py

```diff
@@ -42,21 +41,6 @@ def __init__(
         start_engine_loop: bool = True,
     ) -> None:
 
-        # The child processes will send SIGQUIT when unrecoverable
```

Comment:
NOTE: moved to `CoreClient` so that it can be shared across `AsyncLLM` and `LLMEngine`